CRM/Remoteevent/Localisation.php

```diff
@@ -79,31 +79,67 @@ public function localise($string, $context = [])
     /**
      * Localise a given string with this localisation.
      *
-     * @param string $string
+     * @param string $text
      *   The (English) string to localise.
      *
-     * @param array $context
+     * @param array $params
      *   Localisation parameters or variables.
      *
      * @return string
      *   The localised version of the string.
      *
      * @see \ts()
      */
-    public function ts($string, $context = []) {
-        if (empty($this->locale)) {
-            // No changes, used for pot extraction.
-            return $string;
+    public function ts($text, $params = [])
+    {
+        if (!isset($this->locale)) {
+          // No changes, used for pot extraction.
+          return $text;
+        }
+
+        static $bootstrapReady = FALSE;
```

Comment:
I'd prefer class attributes instead of static method variables.

Reply:
Same here: just copied from Core's `ts()` - I'd consider this approach temporary, as we should `\CRM_Core_I18n::setLocale()` before and after the *CiviRemote Event* request instead, so I think it's ok for now.